frontend/src/pages/Workflows/WorkflowList.js

```diff
@@ -61,36 +61,40 @@ const handleSubmitEdit = props => {
 const getSortableItems = ({ workflowItems, ...props }) => {
   let nextWorkflowNotSelectable = false;
 
-  return workflowItems.map((workflow, index) => {
-    const { displayName, amount, status } = workflow.data;
-    const redacted = displayName === null && amount === null;
-    const currentWorkflowSelectable = !nextWorkflowNotSelectable;
+  return workflowItems.length > 0 ? (
+    workflowItems.map((workflow, index) => {
+      const { displayName, amount, status } = workflow.data;
+      const redacted = displayName === null && amount === null;
+      const currentWorkflowSelectable = !nextWorkflowNotSelectable;
 
-    if (!nextWorkflowNotSelectable) {
-      nextWorkflowNotSelectable = status === "open";
-    }
-    return redacted ? (
-      <RedactedWorkflowItem
-        disabled={!props.workflowSortEnabled || status !== "open"}
-        key={`item-${index}`}
-        index={index}
-        mapIndex={index}
-        workflow={workflow}
-        currentWorkflowSelectable={currentWorkflowSelectable}
-        {...props}
-      />
-    ) : (
-      <WorkflowItem
-        disabled={!props.workflowSortEnabled || status !== "open"}
-        key={`item-${index}`}
-        index={index}
-        mapIndex={index}
-        workflow={workflow}
-        currentWorkflowSelectable={currentWorkflowSelectable}
-        {...props}
-      />
-    );
-  });
+      if (!nextWorkflowNotSelectable) {
+        nextWorkflowNotSelectable = status === "open";
+      }
+      return redacted ? (
+        <RedactedWorkflowItem
+          disabled={!props.workflowSortEnabled || status !== "open"}
+          key={`item-${index}`}
+          index={index}
+          mapIndex={index}
+          workflow={workflow}
+          currentWorkflowSelectable={currentWorkflowSelectable}
+          {...props}
+        />
+      ) : (
+        <WorkflowItem
+          disabled={!props.workflowSortEnabled || status !== "open"}
+          key={`item-${index}`}
+          index={index}
+          mapIndex={index}
+          workflow={workflow}
+          currentWorkflowSelectable={currentWorkflowSelectable}
+          {...props}
+        />
+      );
+    })
+  ) : (
+    <EmptyStates.WorkflowItem />
```

Comment:
this could be done one layer above, i.e. either display the workflow item list or the empty page